src/pages/OtherDashboard/OtherDashboard.vue

```diff
@@ -96,17 +97,35 @@ export default {
           company: "Voyager",
           purchased: false,
           when: "Today",
-          today: true,
-        },
+          today: true
+        }
       ],
+      info: {},
+      positions: []
     };
   },
+  created() {
+    this.initialize();
+  },
+  methods: {
+    async initialize() {
+      this.info = {
+        ...(await UserService.getUserInfo(this.userId)),
```

Comment:
Have you tried the `undefined` value here? I'm not sure if `undefined` could causes error for the `...` operator or not

Reply:
yep it wont cause any errors, I added a render condition if the userId doesn't exist